### ISSUE TITLE
POSH 2.00 remove-hpovlicense

### ISSUE THREAD

**Issue by andrewhpe** [HewlettPackard/POSH-HPOneView#96]
### POSH 1.20 Behavior
PS C:\Users\Administrator> Get-HPOVLicense |Remove-HPOVLicense

Confirm
Are you sure you want to perform this action?
Performing the operation "Remove license from appliance?" on target "HP OneView
 Advanced".
[Y] Yes  [A] Yes to All  [N] No  [L] No to All  [S] Suspend  [?] Help
(default is "Y"):
Successfully removed 'HP OneView Advanced' from appliance

### Actual POSH 2.00 Behavior
PS C:\Users\Administrator>  Get-HPOVlicense | Remove-HPOVLicense
Remove-HPOVLicense : The License resource is not an expected category type [].
 Allowed resource category types are 'licenses'.  Please check the object
provided and try again.
At line:1 char:20
+  Get-HPOVlicense | Remove-HPOVLicense
+                    ~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (License:/rest/l...product/1066
   7-1:PSObject) [Remove-HPOVLicense], InvalidOperationException
    + FullyQualifiedErrorId : InvalidArgumentValue,Remove-HPOVLicense

### Steps to reproduce
1. Add new lic to HPOV 2.0
2. run Get-HPOVlicense | Remove-HPOVLicense immediately after added in step 1

### Version Information
HPE OneView PowerShell Library Version (`Get-HPOVVersion`):
PS C:\Users\Administrator> Get-HPOVVersion
PowerShell Library Version : 2.0.340.0
Library Path               : C:\Users\Administrator\Documents\WindowsPowerShell\Modules\HPOneView.200
HPE OneView Appliance Version:  2.02
Output from `$PSVersionTable` on your Windows Host:

PS C:\Users\Administrator> $PSVersionTable

Name  : PSVersion
Value : 5.0.10514.6

Name  : WSManStackVersion
Value : 3.0

Name  : SerializationVersion
Value : 1.1.0.1

Name  : CLRVersion
Value : 4.0.30319.34209

Name  : BuildVersion
Value : 10.0.10514.6

Name  : PSCompatibleVersions
Value : {1.0, 2.0, 3.0, 4.0...}

Name  : PSRemotingProtocolVersion
Value : 2.3

**Comment by ChrisLynchHPE** [comment]
This is caused by the `Get-HPOVLicense` Cmdlet returning the API License Summary view, which doesn't contain the actual license object.  I will change the default behavior to return license objects, and add a `-Summary` parameter.  This change will fix the error you are getting.  In the meantime, you can use the following as a workaround:

```PowerShell
(Send-HPOVRequest /rest/licenses).members | Remove-HPOVLicense
```

**Comment by ChrisLynchHPE** [comment]
This is fixed in [Release 2.0.368.0](https://github.com/HewlettPackard/POSH-HPOneView/releases/tag/v2.0.368.0).

Closing.
